src/lib.rs

```diff
@@ -457,3 +502,21 @@ macro_rules! lambda {
         lambda! { "handler" => $f, }
     };
 }
+
+
+#[cfg(test)]
+mod tests {
+    use super::LambdaContext;
+    #[test]
+    fn fake_context() {
+        let fake = LambdaContext::fake()
```

Comment:
I assume `LambdaContext::fake()` will be sufficient for most cases there are interfaces for contexts, all which cost nothing in non test contexts